imsim/instcat.py

```diff
@@ -859,4 +825,3 @@ def getKwargs(self, config, base, logger):
 RegisterValueType('InstCatWorldPos', InstCatWorldPos, [CelestialCoord],
                   input_type='instance_catalog')
 RegisterObjectType('InstCatObj', InstCatObj, input_type='instance_catalog')
-RegisterSEDType('InstCatSED', InstCatSEDBuilder(), input_type='instance_catalog')
```

Comment:
I think we probably want to keep this for now, don't we?  I thought we wanted to keep the functionality of using instance catalogs, and this is the way you get an SED with one of those.

Reply:
Yes, of course.  I'll add it back in.